.goreleaser.yml

```diff
@@ -0,0 +1,57 @@
+project_name: nats-server
+
+release:
+  github:
+    owner: nats-io
+    name: nats-server
+  name_template: 'Release {{.Tag}}'
+  draft: true
+
+changelog:
+  skip: true
+
+builds:
+- main: ./main.go
+  binary: nats-server
+  ldflags:
+    - -s -w -X github.com/nats-io/nats-server/server.gitCommit={{.ShortCommit}}
+  env:
+    - GO111MODULE=off
+    - CGO_ENABLED=0
+  goos:
+  - darwin
+  - linux
+  - windows
+  goarch:
+  - amd64
+  - arm
+  - arm64
+  - 386
+  goarm:
+  - 6
+  - 7
+  ignore:
+    - goos: darwin
+      goarch: 386
+
+nfpms:
+  - name_template: '{{.ProjectName}}-{{.Tag}}-{{.Arch}}{{if .Arm}}{{.Arm}}{{end}}'
+    homepage:  https://nats.io
+    description: High-Performance server for NATS, the cloud native messaging system.
+    maintainer: Ivan Kozlovic <ivan@synadia.com>
+    license: Apache 2.0
+    vendor: Synadia Inc.
+    formats:
+    - deb
+    - rpm
+
+archives:
+  - name_template: '{{.ProjectName}}-{{.Tag}}-{{.Os}}-{{.Arch}}{{if .Arm}}{{.Arm}}{{end}}'
+    wrap_in_directory: true
+    format: zip
+    files:
+      - README.md
+      - LICENSE
+
+checksum:
```

Comment:
This satisfy the sha256?

Reply:
Will add the algo in next commit: `algorithm: sha256`

Reply:
So actually sha256 was default, but will still leave this in case that changes. But in order to use rget, I will change the `name_template` to 'SHA256SUMS' following instructions here:
https://github.com/merklecounty/rget/blob/master/Documentation/integrations.md
https://github.com/merklecounty/rget/blob/6c02b9803154ca8d11a68f42afc9894f7e975f4d/.goreleaser.yml#L30

Reply:
ok sounds good. Thanks.